### PR TITLE
P0 homepage

### PR DESCRIPTION
**Pull request by stutisk**
https://user-images.githubusercontent.com/48529714/153743396-bcc4ddb5-b448-418a-a34d-25f4a16dbf8e.mp4

### CODE REVIEW

File: homepage/homepage.css [hunk 1]
```diff
@@ -0,0 +1,146 @@
+:root{
+    --border-color:rgba(94, 91, 91, 0.493);
+    --footer-color:#242729
+}
+
+.span {
+    color: var(--main-Cta-color);
```
Comment (bhanuprakashj) [inline homepage/homepage.css:7]: please change to proper naming convention(kebab case)

File: homepage/homepage.css [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+}
+
+.card-categories {
+    display: flex;
+    flex-direction: row;
+    gap: 3rem;
+    justify-content: center;
+    flex-wrap: wrap;
+    flex: 0 0 1;
+
```
Comment (bhanuprakashj) [inline homepage/homepage.css]: avoid empty lines before closing brace, where ever applicable in rest of the code

File: homepage/homepage.css [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+.footer-container ul {
+    padding: 0;
+    list-style: none;
+    line-height: 1.6;
+    font-size: var(--size-1);
+
+}
+
+.footer-container ul a {
+    color: var(--secondry-color);
```
Comment (bhanuprakashj) [inline homepage/homepage.css:115]: spelling mistake, suggest to change(optional)
Suggestion: you can install "code spell checker" extension in vs code

**Review by bhanuprakashj — COMMENTED** [review]
other than the comments mentioned, code looks good to me!

**Review by GuptaEesh — COMMENTED** [review]
The minor changes are already suggested. 
>UI video shown looks picture perfect. 

Good to go.

File: homepage/homepage.css (hunk 1 above)
Comment (stutisk) [reply homepage/homepage.css:7]: okay will be following it from next time

File: homepage/homepage.css (hunk 2 above)
Comment (stutisk) [reply homepage/homepage.css]: made changes ! thanku

**Pull request merged**